packages/python-packages/doc-warden/warden/WardenConfiguration.py

```diff
@@ -7,6 +7,26 @@
 import os
 
 class WardenConfiguration():
+    REPOSITORY_SETS = {
+        # 0 = groupId
+        'java': { 
+            'URL': 'https://search.maven.org/classic/#search%7Cga%7C1%7Cg:%22[0]%22%20AND%20a:%22{package_id}%22',
```

Comment:
Any reason to not directly link to the package page instead of building the search query?

Reply:
`https://search.maven.org/artifact/<groupId>/<artifactid>`

Reply:
Going to _test_ using `https://oss.sonatype.org/content/repositories/releases/[0]/{package_id}/{package_version}/{package_id}-{package_version}.pom`

Then put a link out to `https://search.maven.org/artifact/{groupId}/{package_id}` in the index itself.

Reply:
[Example](https://github.com/Azure/azure-sdk-for-java/blob/25f95d969b5e103723302b8bbf29d4588a776828/packages.md)

Reply:
We test with one URL (sonatype) and link out to maven central this way now.